README.md

```diff
@@ -1,3 +1,7 @@
+<div align="center">
+  <img src="./doc/nitriding-logo.svg" alt="Nitriding logo" width="250">
+</div>
```

Comment:
I suppose there's no way to centre this without html tags?

Reply:
I haven't come across an alternative.